File: fb.java
public class fb{

  public void fizzbuzz(int n){
    for (int i = 1; i <= n; i++){
  		String result = "";
      String numc=Integer.toString(i);
      if(numc.contains("3")||i % 3 == 0 ){
        result += "Fizz";}
      if(numc.contains("5")||i % 5 == 0){ 
        result += "Buzz";}
     
  		System.out.println( (result == "") ? Integer.toString(i) : result );
    }
	}

	public static void main(String args[]){    
    fb my_fb = new fb();
    my_fb.fizzbuzz(100);
	}
}

File: fb.rb
class FizzBuzz
	def run(n)
		1.upto(n).each do |i|
	    	x = ''
	    	x += "Fizz"  if (i % 3 == 0 or i.to_s.include?("3")) 
	    	x += "Buzz" if (i % 5 == 0 or i.to_s.include?("5"))
	    	puts(x.empty? ? i : x)
	    end
	end
end

myfb = FizzBuzz.new()
myfb.run(100)